.github/workflows/on-merge.yml

```diff
@@ -0,0 +1,51 @@
+name: On merge
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  release-publish:
+    runs-on: ubuntu-latest
+    needs: build
+    steps:
+      - uses: actions/checkout@v2
+      - name: Install dependencies
+        shell: bash
+        run: yarn install --frozen-lockfile
+      - name: Run Build
+        shell: bash
+        run: yarn build
+      - uses: webfactory/ssh-agent@v0.5.4
+        with:
+          ssh-private-key: |
+            ${{ secrets.PERF_BASTION_PRIVATE_KEY }}
```

Comment:
I actually believe these keys are not the correct ones.

Looking at the commands below, the SSH key seems to be used to allow changes created by the changelog to be pushed back to the repo, so the key is most likely this one: `GH_SSH_PRIVATE_KEY_JAHIACI`